### ISSUE TITLE
getcols can return zero, resulting in divide-by-zero

### ISSUE THREAD

**Issue by dannysauer** [TPC-Council/HammerDB#289]
TLDR: It would probably be good if this code either fell back to a sane default of 25x80 or accepted the `$COLUMNS` environment variable (or both), preventing mysterious failures in edge cases. :)  On to the details:

https://github.com/TPC-Council/HammerDB/blob/a86e7f066c3ef40354258dd6e7432308a3fd7044/modules/tclreadline-1.2.tm#L129-L152

I get that the docker container isn't an official image, but when run in Kubernetes interactive mode, getcols can return a terminal width of zero.  That ultimately results in an experience like this:
```
sauer@GRODD:~> kubectl -n danny-yugabyte-demo run --rm -i --tty --restart=Never --image webysther/hammerdb hammerdb-danny -- ./hammerdbcli
HammerDB CLI v3.3
Copyright (C) 2003-2019 Steve Shaw
Type "help" for a list of commands
The xml is well-formed, applying configuration
divide by zero
    while executing
"expr {$cursorLen % $COLUMNS}"
    (procedure "prompt" line 38)
    invoked from within
"prompt $CMDLINE"
    (procedure "tclline" line 125)
    invoked from within
"tclline "
    (procedure "TclReadLine::interact" line 41)
    invoked from within
"TclReadLine::interact"
    invoked from within
"if { $autostart::autostartap == "true" } {
	source $autostart::autoloadscript
	} else {
    TclReadLine::interact
	}"
    (file "./hammerdbcli" line 84)
pod "hammerdb-danny" deleted
pod danny-yugabyte-demo/hammerdb-danny terminated (Error)
```

Running the same command, but executing bash and then running hammerdbcli is perfectly happy:
```
sauer@GRODD:~> kubectl -n danny-yugabyte-demo run --rm -i --tty --restart=Never --image webysther/hammerdb hammerdb-danny -- /bin/bash
If you don't see a command prompt, try pressing enter.
root@hammerdb-danny:/hammerdb# ./hammerdbcli 
HammerDB CLI v3.3
Copyright (C) 2003-2019 Steve Shaw
Type "help" for a list of commands
The xml is well-formed, applying configuration
hammerdb>
```

The terminal definitely knows how wide it is:
```
sauer@GRODD:~> kubectl -n danny-yugabyte-demo run --rm -i --tty --restart=Never --image webysther/hammerdb hammerdb-danny -- tput cols
80
pod "hammerdb-danny" deleted
```

But `stty` doesn't:
```
sauer@GRODD:~> kubectl -n danny-yugabyte-demo run --rm -i --tty --restart=Never --image webysther/hammerdb hammerdb-danny -- stty size
0 0
pod "hammerdb-danny" deleted
```

This works inside the bash shell because /etc/bash.bashrc runs `shopt -s checkwinsize`. :/
```
sauer@GRODD:~> kubectl -n danny-yugabyte-demo run --rm -i --tty --restart=Never --image webysther/hammerdb hammerdb-danny -- /bin/bash
If you don't see a command prompt, try pressing enter.
root@hammerdb-danny:/hammerdb# stty size
42 80
root@hammerdb-danny:/hammerdb# exit
exit
pod "hammerdb-danny" deleted
```

I'll open an issue against the container as well to fix that issue.  But that's just one container and not really the root cause - which is addressed in the TLDR at the top of the issue. :)

**Comment by sm-shaw** [comment]
Many thanks for the issue - yes, the docker container is not officially released or tested by this team, however still very glad to see it and packaging is becoming a key issue including an updated container release we can support. This is a clear explanation and straightforward to fix, so it makes sense to update it for the next release.
